cmd/metadata-server/server.go

```diff
@@ -93,6 +127,62 @@ func (s *Server) PutFile(ctx context.Context, req *mpb.PutFileRequest) (*mpb.Put
 	return &mpb.PutFileResponse{}, nil
 }
 
+func (s *Server) PutFileStream(stream mpb.MetadataService_PutFileStreamServer) error {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+
+	ctx := stream.Context()
+	storeStream, err := s.storageClient.PutFileStream(ctx)
+	if err != nil {
+		return err
+	}
+
+	in, err := stream.Recv()
+	if err != nil {
+		return err
+	}
+	// First message determines the key and metadata, it will be assumed that all subsequent
+	// file keys are the same
+	key := in.Key
+	if err = storeStream.Send(&spb.PutFileStreamRequest{Key: key, FileChunk: in.FileChunk}); err != nil {
+		return err
+	}
+	entry := in.Metadata
+	if entry == nil {
+		return errors.New("empty metadata")
+	}
+	fileSize := len(in.FileChunk)
```

Comment:
Just realized this variable isn't being used anymore since we get it from FUSE now